README.md

```diff
@@ -51,7 +51,21 @@ Contains the JSON schemas used to validate the sources in the integration tests.
 
 ## GTFS Realtime Data Structure
 
-Still in progress. [Please review the working document to provide your feedback](https://docs.google.com/document/d/1Mlz3AXHItInitsOEAKKi8hZV9d3t3gpFbyIZGm67ESU/edit#heading=h.88f252jw0urf).
+|     Field Name     |  Required from users  |                                                                              Definition                                                                             |
+|:------------------:|:---------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------:
+| MDB Source ID      | No - system generated | Unique numerical identifier.      |   |   |
+| Data Type          | Yes                   | The data format that the source uses: GTFS.                                                                                                            |   |   |
+| Country Code       | Yes                   | ISO 3166-1 alpha-2 code designating the country where the system is located. For a list of valid codes [see here](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes).                                                    |   |   |
```

Comment:
I'm not sure if we should add the static_reference's attributes here as they won't be in the JSON files. Maybe we should add the CSV columns in another section (GTFS + GTFS RT attributes)?